server/welcomebot.go

```diff
@@ -177,6 +200,17 @@ func (p *Plugin) renderWelcomeMessage(messageTemplate MessageTemplate, configMes
 	return post
 }
 
+func (p *Plugin) getTemplateMessage(data []string, messageTemplate interface{}) string {
+	tmpMsg, _ := template.New("Response").Parse(strings.Join(data, "\n"))
+	var message bytes.Buffer
+	if appErr := tmpMsg.Execute(&message, messageTemplate); appErr != nil {
```

Comment:
I think this is a regular error right?

```suggestion
	if err := tmpMsg.Execute(&message, messageTemplate); err != nil {
```